### PR TITLE
Minor fixes related to openEPD EPD -> Pydantic refactoring

### PR DESCRIPTION
**Pull request by alexche-cchange**

### CODE REVIEW

File: src/openepd/model/common.py [hunk 1]
```diff
@@ -117,6 +117,18 @@ class Ingredient(BaseOpenEpdSchema):
     )
     citation: str | None = pyd.Field(default=None, description="Text citation describing the data source ")
 
+    @pyd.root_validator(skip_on_failure=True)
+    def _validate_evidence(cls, values: dict[str, Any]) -> dict[str, Any]:
+        # gwp_fraction should be backed by some type of evidence (fraction coming from product EPD etc) to be accounted
+        # for in the calculation of uncertainty
+        if values.get("gwp_fraction"):
```
Comment (alexlitvinenko) [inline src/openepd/model/common.py:124]: Missing "gwp_fraction" key and `None` value means different things when `PATCH` request. Are you taking this into account?

File: src/openepd/model/specs/singular/concrete.py [hunk 2]
```diff
@@ -183,3 +183,16 @@ class ConcreteV1(BaseOpenEpdHierarchicalSpec):
     _csa_exposure_classes_exclusive_groups_validator = pyd.validator("csa_exposure_classes", allow_reuse=True)(
         exclusive_groups_validator_factory(CsaExposureClass)
     )
+
+    @pyd.root_validator(skip_on_failure=True)
+    def _validate_strength_other(cls, values: dict) -> dict:
+        strength_other = values.get("strength_other")
```
Comment (alexlitvinenko) [inline src/openepd/model/specs/singular/concrete.py:189]: Missing "strength_other" key and `None` value means different things when `PATCH` request. Are you taking this into account?

**Review by alexlitvinenko — APPROVED** [review]

File: src/openepd/model/specs/singular/concrete.py (hunk 2 above)
Comment (alexche-cchange) [reply src/openepd/model/specs/singular/concrete.py:189]: uh, okay. I perceived the `strength_other` and `strength_other_d` as a single field by meaning, and would expect this to come always in pairs. 

However, if only one should be supplied, I will move this to the other level and validate at the object before the update (ensure that invariant holds that if strength then days).

File: src/openepd/model/common.py (hunk 1 above)
Comment (alexche-cchange) [reply src/openepd/model/common.py:124]: We pass Ingredients as an array, and there is no identifier there, so we are only able to replace the entire ingredients array and not patch them.

**Review by alexlitvinenko — APPROVED** [review]

**Pull request merged**